public/js/controllers/coding_progress_table_controller.js

```diff
@@ -138,9 +209,20 @@ export class CodingProgressTableController {
         return [k, v];
     }
 
-    static jsonToArray(json) {
+    static jsonToArray(json, tableSection="") {
         let arr = [];
         for (const key in json) {
+            // Filter columns
+            if (CodingProgressTableController.selectedColumns.includes(key)) { continue }
+            // Filter rows
+            if (tableSection == "td") {
+                if (CodingProgressTableController.keyword != "") {
+                    const terms = CodingProgressTableController.keyword.split(/_|-| /).map(v => v.toLowerCase()),
```

Comment:
for active projects, let us match the exact keyword. e.g COVID19 should match only datasets that match that exact word.